### PR TITLE
doc: add man page for ceph-create-keys

### PR DESCRIPTION
**Pull request by tchaikov**
* add man page for ceps-create-keys
* update ceph-conf.8

### CODE REVIEW

File: doc/man/8/ceph-conf.rst [hunk 1]
```diff
@@ -28,30 +28,59 @@ Ceph configuration file to use with the ``-c`` flag.
...
+.. option:: --name *type.id*
+
+   the Ceph name in which the sections are searched (default 'client.admin').
+   For example, if we specify ``--name osd.0``, the following sections will be
+   searched: [osd.0], [osd], [global]
+
+.. option:: -r, --resolve-search
+
+   search for the first file that exists and can be opened in the resulted
+   comman delimited search list.
```
Comment (trhoden) [inline doc/man/8/ceph-conf.rst]: s/comma/comman

Comment (tchaikov) [inline doc/man/8/ceph-conf.rst]: thanks for your review @trhoden . it is fixed in latest patch.

**Comment by tchaikov** [comment]
thanks for your review @dachary , i just realized that i need to put the rst file into the dist tarball! will prepare a pull request shortly.

**Comment by dachary** [comment]
@tchaikov I missed that :-(

**Comment by tchaikov** [comment]
sorry for my mistake. should always create a wip branch whenever i touch our building system ! i posted a fix to #4655 .

**Pull request merged**